src/main/java/core/basesyntax/Reader.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class Reader {
```

Comment:
It was enough to simply divide by methods; there is no need to divide into classes.